### PR TITLE
Add default language check

### PR DESCRIPTION
**Pull request by cannc4**
Added default language check to parseResult to correctly handle the empty language prop

### CODE REVIEW

**Comment by cannc4** [comment]
Hi @Mattchewone, could you please merge this if everything looks fine?

File: lib/parse-result.js [hunk 1]
```diff
@@ -15,6 +15,8 @@ module.exports = function (options) {
       throw new errors.Forbidden('`parseI18nResult` can only be run on a `after` hook.')
     }
 
+    if (!options.language) options.language = 'en'
```
Comment (Mattchewone) [inline lib/parse-result.js]: I don't think this is needed, as the `options = Object.assign({}, defaults, options)` would assign a default.

File: lib/parse-result.js [hunk 2]
```diff
@@ -33,15 +35,15 @@ function parseResult (result, language, props = []) {
     return null
   }
 
-  const data = [].concat(result.data || result)
+  let data = [].concat(result.data || result)
```
Comment (Mattchewone) [inline lib/parse-result.js]: This can still be a `const`, shouldn't need to change this

File: lib/parse-result.js [hunk 3]
```diff
@@ -33,15 +35,15 @@ function parseResult (result, language, props = []) {
...
 
   data.forEach((item, index) => {
     Object.keys(item)
       .forEach(key => {
         if (props.includes(key)) {
           // Find data of specified language, otherwise fallback to default
-          const res = data[index][key][language] || data[index][key][defaults.language]
-          if (res) {
+          let res = data[index][key][language] || data[index][key][defaults.language]
```
Comment (Mattchewone) [inline lib/parse-result.js]: Same here, this can also still be a `const` we are not re-assigning `res`.

**Comment by cannc4** [comment]
Ah yes, did clean up some things! some were left from the previous issue that we had a few weeks ago.

**Review by Mattchewone — APPROVED** [review]

**Pull request merged**